ext/winevt/winevt_bookmark.c

```diff
@@ -68,6 +93,12 @@ rb_winevt_bookmark_initialize(int argc, VALUE* argv, VALUE self)
   return Qnil;
 }
 
+/*
+ * This function updates bookmark and returns Bookmark instance.
```

Comment:
nitpick: The term `function` seems inappropriate for Ruby, use `method` instead.

Reply:
Oh!